tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/Net/GraphQLCommon.cs

```diff
@@ -184,5 +191,129 @@ private static string ConstructErrorMessage(IExecutionErrors executionErrors)
+        private static List<SpanEvent> ConstructErrorEvents(IExecutionErrors executionErrors)
+        {
+            List<SpanEvent> spanEvents = [];
+
+            try
+            {
+                for (int i = 0; i < executionErrors.Count; i++)
+                {
+                    var eventAttributes = new List<KeyValuePair<string, object>>();
+                    var executionError = executionErrors[i];
+
+                    if (executionErrors[i].Instance.TryDuckCast<IExecutionErrorExtensions>(out var executionErrorExtensions))
+                    {
+                        var extensions = executionErrorExtensions.Extensions;
+                        if (extensions is { Count: > 0 })
+                        {
+                            var configuredExtensions = Tracer.Instance.Settings.GraphQLErrorExtensions;
+
+                            foreach (var extension in extensions)
+                            {
+                                if (configuredExtensions.Contains(extension.Key))
+                                {
+                                    var key = extension.Key;
+                                    var value = extension.Value;
+
+                                    if (value == null)
+                                    {
+                                        value = "null";
+                                    }
+                                    else if (value is Array array)
+                                    {
+                                        var builder = Util.StringBuilderCache.Acquire();
+
+                                        try
+                                        {
+                                            builder.Append('[');
+                                            for (var k = 0; k < array.Length; k++)
+                                            {
+                                                var item = array.GetValue(k);
+                                                if (k > 0)
+                                                {
+                                                    builder.Append(',');
+                                                }
+
+                                                builder.Append(item?.ToString() ?? "null");
+                                            }
+
+                                            builder.Append(']');
+                                            value = Util.StringBuilderCache.GetStringAndRelease(builder);
+                                        }
+                                        catch (Exception ex)
+                                        {
+                                            Console.WriteLine(ex);
+                                            Util.StringBuilderCache.Release(builder);
+                                        }
+                                    }
+                                    else if (value is ulong or decimal || !Util.SpanEventConverter.IsAllowedType(value))
```

Comment:
nit: The checks for `ulong` and `decimal` seem unnecessary to me, as they are already covered by the `!Util.SpanEventConverter.IsAllowedType(value)` check, right?

Reply:
We removed them in the previous PR to align better with it:
https://github.com/DataDog/dd-trace-dotnet/blob/31ab4370841727380acbec826f28bd2d72d8f0e5/tracer/src/Datadog.Trace/Util/SpanEventConverter.cs#L87C1-L88C1